<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>264</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_ID">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>230</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:transparent;
color:black;
}

QPushButton:hover{
font:bold;
}</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PSW">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:transparent;
color:black;
}

QPushButton:hover{
font:bold;
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>230</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:black;}</string>
    </property>
    <property name="text">
     <string>or</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:black;}</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_id">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
padding-bottom:7px;
color:black;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>User ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:transparent;
color:black;
}

QPushButton:hover{
font:bold;
}</string>
    </property>
    <property name="text">
     <string>sign up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
padding-bottom:7px;
color:black;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid;
border-radius:15px;
color:black;
}

QPushButton:hover
{
background-color: white;

}
QPushButton:pressed
{
background-color:rgb(204, 204, 204);

}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:black;}</string>
    </property>
    <property name="text">
     <string>Forgot your</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>true</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:black;}</string>
    </property>
    <property name="text">
     <string>KAI</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_errormsg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px;
color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>pushButton_ID</zorder>
   <zorder>pushButton_PSW</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>login_id</zorder>
   <zorder>signup</zorder>
   <zorder>login_password</zorder>
   <zorder>login</zorder>
   <zorder>label_2</zorder>
   <zorder>login_errormsg</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../../../Applications/test.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>start()</slot>
 </slots>
</ui>
